src/context/RecipesContext.js

```diff
@@ -1,28 +1,38 @@
-import React, { createContext, useContext } from 'react';
+import React, { createContext, useContext, useState } from 'react';
 import PropTypes from 'prop-types';
 
-export const RecipesContext = createContext({
-  meals: {
-    categories: [], // Lista de categorias recuperadas pela API
-    list: [], // Lista de comidas recuperadas pela API
-    inProgress: {}, // Objeto onde cada chave é o id da receita em andamento e o valor correspondente é o array com os ingredientes já marcados
-    favorites: [], // Lista de receitas favoritas
-    finished: [], // Lista de receitas finalizadas
-  },
-  cocktails: {
-    categories: [], // Lista de categorias recuperadas pela API
-    list: [], // Lista de comidas recuperadas pela API
-    inProgress: {}, // Objeto onde cada chave é o id da receita em andamento e o valor correspondente é o array com os ingredientes já marcados
-    favorites: [], // Lista de receitas favoritas
-    finished: [], // Lista de receitas finalizadas
-  },
-});
+// {
+//   meals: {
+//     categories: [], // Lista de categorias recuperadas pela API
+//     list: [], // Lista de comidas recuperadas pela API
+//     inProgress: {}, // Objeto onde cada chave é o id da receita em andamento e o valor correspondente é o array com os ingredientes já marcados
+//     favorites: [], // Lista de receitas favoritas
+//     finished: [], // Lista de receitas finalizadas
+//   },
+//   cocktails: {
+//     categories: [], // Lista de categorias recuperadas pela API
+//     list: [], // Lista de comidas recuperadas pela API
+//     inProgress: {}, // Objeto onde cada chave é o id da receita em andamento e o valor correspondente é o array com os ingredientes já marcados
+//     favorites: [], // Lista de receitas favoritas
+//     finished: [], // Lista de receitas finalizadas
+//   },
+// }
 
-export const RecipesProvider = ({ children }) => (
-  <RecipesContext.Provider>
-    { children }
-  </RecipesContext.Provider>
-);
+export const RecipesContext = createContext();
+
+export const RecipesProvider = ({ children }) => {
+  const [list, setList] = useState([]);
+
+  const getRecipesList = (recipesList) => {
```

Comment:
```suggestion
  const setRecipesList = (recipesList) => {
```
se a função está gravando algo ela deveriar ser um set. Get são funções que leem algo.